src/SIL.XForge.Scripture/ClientApp/src/material-styles.scss

```diff
@@ -121,7 +123,7 @@ $material-theme-accent-error: mat.define-light-theme(
 @include mat.card-theme($material-app-theme);
 @include mat.checkbox-theme($material-app-theme);
 @include mat.chips-theme($material-app-theme);
-@include mat.legacy-dialog-theme($material-app-theme);
+@include mat.dialog-theme($material-app-theme);
```

Comment:
The text on the dialog ends up looking very faded. I think this is controlled by the theme we pass here. We may need to adjust the theme we pass here so the text color isn't so light. It's much darker in the examples at https://material.angular.io/components/dialog/examples

Also, please make sure you're reading through the upgrade information at https://material.angular.io/guide/mdc-migration#dialog

![](https://github.com/sillsdev/web-xforge/assets/6140710/f04821d1-0236-40f0-a1a4-a3bd42a3e749)